index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+      <nav>
+        <div class="logo">
+          <h4>My logo</h4>
```

Comment:
Because this requirement was not mentioned by the first reviewer, please take this requirement as optional.

As mentioned by the reviewer, it is advisable to have the logo as a link to the top of the page. In order to implement this, an anchor is recommended to be used (`a` tag).